makefiles/Makefile.docs

```diff
@@ -1,16 +1,23 @@
 GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
 USER=$(shell whoami)
+
+SNOOTY_ENV = $(shell printenv SNOOTY_ENV)
+
 STAGING_URL="https://docs-mongodborg-staging.corp.mongodb.com"
-STAGING_BUCKET=docs-mongodb-org-staging
-
-ifeq ($(SNOOTY_INTEGRATION),false)
-  PRODUCTION_URL="https://docs.mongodb.com"
-  PRODUCTION_BUCKET=docs-mongodb-org-prod
-else
-  PRODUCTION_URL="docs-mongodbcom-integration.corp.mongodb.com"
-  PRODUCTION_BUCKET=docs-manual-integration
+STAGING_BUCKET=docs-mongodb-org-stg
+
+ifeq ($(SNOOTY_ENV), production) 
```

Comment:
Would it make sense to put this bit into shared.mk for all of the `docs.mongodb.com/` sites with a boolean flag to skip it for the other sites?

Reply:
moved it